tests/integration_test.rs

```diff
@@ -1 +1,4 @@
-
+#[test]
+fn test_add() {
+    assert!(true);
+}
```

Comment:
Still WIP?

Reply:
Yep. No integration tests, yet, but the file can't be empty.